DOCUMENTATION.md

```diff
@@ -368,6 +368,68 @@ If the request is not successful, the client returns an `HTTPError` containing t
 |`500`|`[{`<br>`"error": "Exception",`<br>`"message": "Internal server error"`<br>`}]`|Notify was unable to process the request, resend your notification.|
 
 
+### Send a precompiled Letter
+This is an invitation-only feature. Contact the GOV.UK Notify team on the support page or through the Slack channel for more information.
```

Comment:
Could add links to our support page and the cross-gov Slack channel here to keep it consistent with the Java docs